commands/build.go

```diff
@@ -81,10 +81,9 @@ type commonOptions struct {
 	progress     string
 	pull         *bool
 
-	// golangci-lint#826
-	// nolint:structcheck
+	//nolint:structcheck
 	exportPush bool
-	// nolint:structcheck
+	//nolint:structcheck
```

Comment:
These should no longer be needed if we disable the linter? Or is it still running in the default set? (if so, probably an update to 1.49 or 1.50 is needed); https://github.com/docker/buildx/blob/1bb375fe5ccad33244bcc3ede169e1cabcc7fcc1/hack/dockerfiles/lint.Dockerfile#L7

Reply:
Yikes, I screwed up the commit there, I could have *sworn* I removed that line :thinking:

Anyways, have properly removed them now.